packages/connect-core/src/connect-create-request-header.ts

```diff
@@ -33,5 +35,6 @@ export function connectCreateRequestHeader(
   if (timeoutMs !== undefined) {
     result.set("Connect-Timeout-Ms", `${timeoutMs}`);
   }
+  result.set("connect-protocol-version", protocolVersion);
```

Comment:
Can you use uppercase starting letters, as in `Connect-Protocol-Version`, like we do for other fields?